src/internet_identity/internet_identity.did

```diff
@@ -122,10 +122,22 @@ type GetDelegationResponse = variant {
 type InternetIdentityStats = record {
   users_registered: nat64;
   assigned_user_number_range: record { nat64; nat64; };
+  archive_info: ArchiveInfo;
+};
+
+// Information about the archive.
+type ArchiveInfo = record {
```

Comment:
Yes, and both are valid states:
* `archive_canister: none, expected_wasm_hash: some`: the archive has not yet been deployed, but it could be, by calling `deploy_archive` with a wasm module matching the hash.
* `archive_canister: some, expected_wasm_hash: none`: although currently unsupported (due to the way init arguments are set up) this would just mean that there is an archive, but we do not allow any changes to it (thus there is no valid wasm hash for redeployment).